src/modules/network/hooks/queries/useNetworkStatus.js

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@tanstack/react-query';
+import { useCurrentApplication } from '@blockchainApplication/manage/hooks';
+import { NETWORK_STATUS, APPLICATION } from 'src/const/queries';
+import {
+  METHOD,
+  API_VERSION,
+  API_METHOD,
+} from 'src/const/config';
+
+// eslint-disable-next-line import/prefer-default-export
+export const useNetworkStatus = ({ config: customConfig = {}, options } = { }) => {
+  const [currentApplication] = useCurrentApplication();
+  const config = {
+    baseUrl: currentApplication?.apis[0][METHOD] ?? currentApplication?.apis[0].rest,
+    path: `/api/${API_VERSION}/network/status/`,
```

Comment:
@soroushm The last slash can be removed because if a search param is added, it might look like `/api/v3/network/status/?node=xyz`